### ISSUE TITLE
Support Local account by disable MS Account on win11

### ISSUE THREAD

**Issue by dtantono** [WoeUSB/WoeUSB-ng#88]
Hi
I know Rufus have an ability to allow Local account installation on windows11. But, that software is windows only. So, I hope this app have same ability like that.

**Comment by WaxyMocha** [comment]
No, this software just creates bootable inhalators for windows.
